Sources/Rendering/Core/Prop3D/index.js

```diff
@@ -88,7 +88,23 @@ function vtkProp3D(publicAPI, model) {
   };
 
   publicAPI.SetUserTransform = notImplemented('SetUserTransform');
-  publicAPI.SetUserMatrix = notImplemented('SetUserMatrix');
+
+  publicAPI.setUserMatrix = (matrix) => {
+    model.isIdentity = false;
+    model.userMatrix = null;
+    if (!matrix) {
+      return;
+    }
+
```

Comment:
Not sure to understand. Can you explain again?

Reply:
Sure, 

If someone sets the userMatrix then you mark isIdentity as false. Assuming that the user Matrix is not identity. That is fine. No problems. But if they then set the userMatrix back to null you still assume that the actor is not identity. I think it would be better in that case (setting the userMatrix to null) to check if the actor's matrix is identity or not and set the flag accordingly.